sdk/identity/azure-identity/vcpkg/vcpkg.json

```diff
@@ -18,7 +18,7 @@
     {
       "name": "azure-core-cpp",
       "default-features": false,
-      "version>=": "1.9.0"
+      "version>=": "1.14.0-beta.1"
```

Comment:
We have to be careful not to ship a GA version of these packages with a beta core dependency (also storage/etc.). Just calling it out to keep on top of mind.

The upcoming release of identity (this week) is a beta, so this is OK.

Reply:
We won't be able to ship a GA version of the packages with a beta core dependency - they will fail at the vcpkg publish step since the beta versions are not present in the vcpkg main repository.